src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -209,3 +209,52 @@ export const additionalDocumentDescription = () => {
   );
 };
 
+const documentLabels = {
+  1: 'Discharge',
+  2: 'Marriage related',
+  3: 'Dependent related',
+  // 4: 'VA preneed form',
+  5: 'Letter',
+  6: 'Other'
+};
+
+const getVACenterName = (center) => center.treatment.treatmentCenterName;
+const getPrivateCenterName = (release) => release.privateRecordRelease.treatmentCenterName;
+
+const listifyCenters = (center, idx, list) => {
+  const getName = center.treatment ? getVACenterName : getPrivateCenterName;
+  const notLast = idx < (list.length - 1);
+  const justOne = list.length === 1;
+  const atLeastThree = list.length > 2;
+  return (
+    <strong key={idx}>
+      {`${notLast || justOne ? '' : ' and '}${getName(center)}${atLeastThree && notLast ? ', ' : ''}`}</strong>
```

Comment:
Is there a benefit to calling `getName()` in here rather than in the block of `const`s above?
Also, can we move that `</strong>` down a line?
Also, I approve the use of the Oxford comma. :sunglasses:

Reply:
Moving the `getName` call up was the right call, updated.